OracleSOABPM/dockerfiles/README.md

```diff
@@ -0,0 +1,33 @@
+How to Build the image
+======================
+Before you Start
+----------------
+1. **FIRST TIME ONLY**
+   - Update the OracleSOABPM/setenv.sh file to point
+     to the right locations for the following environment variables
+     specific to your environment (See top of file)    
+     DC_USERHOME, DC_REGISTRY_SOA, DC_REGISTRY_DB    
+   - Update the OracleSOABPM/setenv.sh file with the proxy details
+     if you have an internal proxy - ensure that the proxy
+     related env variables are set    
+     http_proxy, https_proxy, no_proxy
+   - Pull the base image into your local docker cache for builds.
```

Comment:
The preferred mechanism for manually building images is to manually build the base image. See the WebLogic Dockerfiles for examples on how to either manually build the `oracle/serverjre` image or pull it from the Oracle Container Registry or Docker Store and tag it accordingly.